src/components/dialogs/two-windings-transformer/ratio-tap-changer-pane.js

```diff
@@ -235,6 +218,12 @@ const RatioTapChangerPane = (props) => {
 
     const handleEditCell = useCallback(
         (rowData, newVal) => {
+            setLineEdit({
+                oldValues: {},
+                newValues: {},
+                id: rowData.rowIndex,
+                errors: new Map(),
+            });
             const parsedVal = parseInt(newVal);
```

Comment:
Same as with the phase tap pane, parseInt prevent to submit decimal values while editing ratio field, maybe change it to parseFloat ?

Reply:
done.